include/ensmallen_bits/nsga2/nsga2.hpp

```diff
@@ -177,6 +178,25 @@ class NSGA2 {
   //! `Optimize()` has been called.
   const arma::cube& ParetoFront() const { return paretoFront; }
 
+  /**
+   * Retrieve the best front (the Pareto frontier).  This returns an empty
+   * vector until `Optimize()` has been called.  Note that this function is
+   * deprecated and will be removed in ensmallen 3.10.0!  Use `ParetoFront()` instead.
```

Comment:
```suggestion
   * deprecated and will be removed in ensmallen 3.0.0!  Use `ParetoFront()` instead.
```

We're still on the 2.x track now...

Reply:
We've been starting the minor versions with `.10`, a bit like Armadillo, so that versions sort correctly numerically too. :)  Anyway the issue can be avoided by just using `3.x`... :)  I made that change in 9906c22d. :+1:

Reply:
@rcurtin missed that. Thanks for the clarification!